iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerLexicalGrammar.java

```diff
@@ -97,6 +100,7 @@ private static void lexical(LexerlessGrammarBuilder b) {
     b.rule(STRING_LITERAL_WITHOUT_SPACE).is(b.regexp(DockerLexicalConstant.STRING_LITERAL));
     b.rule(STRING_UNTIL_EOL).is(SPACING, b.regexp(DockerLexicalConstant.STRING_UNTIL_EOL));
     b.rule(EQUALS_OPERATOR).is(b.regexp(DockerLexicalConstant.EQUALS_OPERATOR));
+    b.rule(PLATFORM).is(Punctuator.MINUS, Punctuator.MINUS, b.regexp("platform"));
```

Comment:
If it is just simple text `platform` without any Regex features, can be here just a simple String `"platform"` instead of regexp call?

Reply:
Unfortunately, the LexerlessGrammarBuilder does not provide any method to simply identify a String.